package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Untitled Dinosaur Game Online - Backend",
   "main": "src/index.js",
   "scripts": {
-    "start": "node src/index.js",
+    "start": "nodemon src/index.js",
```

Comment:
```suggestion
    "start": "node src/index.js",
```

Reply:
I added "dev" command which uses nodemon by default, so we can do `npm run dev` instead